streamr-dev/streamr-platform
# [PLATFORM-1007] No login required for public canvases

* Removes hard login requirement for public canvases in the Canvas editor.
* Shows embed view if totally logged out.
* Adds new "embed toolbar" in bottom left corner of embedded canvases.

![image](https://user-images.githubusercontent.com/43438/63329716-0be23c80-c365-11e9-9336-c71fac1df694.png)

## To Test

1. Share a canvas publicly.
2. Copy canvas url, access via new incognito window.
3. Canvas should be visible in "embed mode" (should look similar to screenshot above).
4. Click "view on Streamr" button in bottom left corner.
5. Should take you to login, and redirect to the canvas in read-only mode (not embedded mode) after login.
